examples/cdn-styles/index.html

```diff
@@ -1,255 +1,473 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Polaris CDN Styles</title>
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@shopify/polaris@4.26.1/styles.min.css"
```

Comment:
```suggestion
      href="https://unpkg.com/@shopify/polaris/styles.min.css"
```

Reply:
This is gonna make the example more brittle, as we publish changes that are breaking when you're using CSS styles only in minor versions (e.g. adding a wrapping div somewhere).

Reply:
I agree, pointing to a specific version makes these examples less likely to unexpectedly break.